src/assets/css/responsive.scss

```diff
@@ -17,6 +17,7 @@ $sub-black-color: #111111;
 $transition: .5s;
 $font-size: 14px;
 
+// Start of critical mobile styles

```

Comment:
Instead of doing comments here, you can wrap this around a region -> `// #region` `// #endregion`

Reference [documentation](https://code.visualstudio.com/docs/editor/codebasics#_folding) if you need to.